openshift/oadp-operator
# fix: Add table for badges in README

## Why the changes were made

Follow up of https://github.com/openshift/oadp-operator/pull/1560 to address https://github.com/openshift/oadp-operator/pull/1560#discussion_r1801797759

## How to test the changes made

To review check https://github.com/mateusoliveira43/oadp-operator/blob/fix/markdown-table/README.md

Check if information is easier to read.
